src/qibo/tests_new/test_core_gates.py

```diff
@@ -252,11 +241,11 @@ def test_cun(backend, name, params):
     qibo.set_backend(original_backend)
 
 
-def test_czpow(backend):
+def test_cu1(backend):
```

Comment:
I think this test is now redudant because `cu1` is tested in `test_cun` which is right above this one anyway.